player/lua/stats.lua

```diff
@@ -641,13 +643,26 @@ local function add_header(s)
 end
 
 
-local function add_file(s, print_cache)
+local function add_file(s, print_cache, print_tags)
     append(s, "", {prefix="File:", nl="", indent=""})
     append_property(s, "filename", {prefix_sep="", nl="", indent=""})
     if mp.get_property_osd("filename") ~= mp.get_property_osd("media-title") then
         append_property(s, "media-title", {prefix="Title:"})
     end
 
+    if print_tags then
+        local tags = mp.get_property_native("display-tags")
+        local tags_displayed = 0
+        for _, tag in ipairs(tags) do
+            local value = mp.get_property_native("metadata/" .. tag)
```

Comment:
hmm, documentation says. Could we switch to `metadata/by-key/`? Also maybe instead of native, use `get_property`?

```
metadata/<key>
    Old version of metadata/by-key/<key>. Use is discouraged, because the metadata key string could conflict with other sub-properties.
```